portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/Comments.path

```diff
@@ -115,7 +115,7 @@
 </tr>
 <tr>
 	<td>USER_NAME</td>
-	<td>//header[@class='lfr-discussion-message-author']/a | //header[@class='lfr-discussion-message-author']//span[contains(.,'${key_userFullName}')]</td>
+	<td>//header[@class='lfr-discussion-message-author']/a | //header[@class='lfr-discussion-message-author']//span[contains(.,'${key_userFullName}')] | //article[contains(@class,'lfr-discussion')]//a[contains(@class,'username')]</td>
```

Comment:
@pedroqueiroz why doesn't this match our existing paths for comments? Is there some difference in the way it's implemented in Calendar?

If it's not worth fixing, I would say make your own path in a Calendar path file, unless @patriciajeanperez has a different opinion.

Reply:
I can take a look on it, I didn't really went through the implementation (it's not what this ticket is about). I just fixed the path because it's been failing for a long time.

I will wait on @patriciajeanperez opinion to create the new path for Calendar.

Reply:
sorry, to clarify, we're asking @patriciajeanperez if she thinks this fix is appropriate, or if Pedro should create his own path in a separate file.

Reply:
Hi @pedroqueiroz and @jpince! I checked and saw that there are similar assertions we can make for username in Calendar and Blogs comment, so I'm ok with having Calendar and Blogs tests use this same locator. We can push this fix for now, and I will take note to refactor this locator to avoid too many ORs. :)

Reply:
@patriciajeanperez thanks!